rollup.config.js

```diff
@@ -36,18 +38,14 @@ const config = {
       runtimeHelpers: true,
       ...babelConfig,
     }),
+    format === 'umd' && globals(),
+    env === 'production' &&
+      uglify({
+        compress: {
+          warnings: false,
+        },
+      }),
     filesize(),
-  ],
+  ].filter(Boolean),
```

Comment:
@tdeekens thanks for the JavaScript lesson - I assume this part is removing empty  / undefined elements in the array?

Reply:
Yup. `[undefined, null, 0].filter(Boolean)` => `[0]`. Essentially it removes any 'nil`ed or holes from the array.